infrastructure/tooling/src/release/tasks.js

```diff
@@ -396,8 +396,8 @@ module.exports = ({tasks, cmdOptions, credentials, baseDir}) => {
     ensureTask(tasks, {
       title: `Publish ${clientName} to npm`,
       requires: [
-        'version-updated',
-        'target-monoimage', // to make sure the build succeeds first..
+        //'version-updated',
+        //'target-monoimage', // to make sure the build succeeds first..
```

Comment:
Intentional commenting?

Reply:
nope, I think I forgot to save after testing :)